docs/source/interfaces/isis_sans/Runs Tab.rst

```diff
@@ -109,17 +109,26 @@ Table Columns
 |                          |   **WavelengthMin**, **WavelengthMax**, **EventSlices** (see :ref:`ISIS_SANS_Settings_Tab-ref`  |
 |                          |   for details), **BackgroundWorkspace**, and **ScaleFactor** can be set (see below).            |
 +--------------------------+-------------------------------------------------------------------------------------------------+
+| **Background Workspace** |   This column allows the user to provide a workspace for scaled background subtraction          |
+|                          |   (see below).                                                                                  |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **Scale Factor**         |   Scale factor to be used for scaled background subtraction (see below).                        |
++--------------------------+-------------------------------------------------------------------------------------------------+
 
 .. _ISIS_SANS_scaled_background-ref:
 
 Scaled Background Subtraction
 +++++++++++++++++++++++++++++
 
-By setting **both** ``BackgroundWorkspace`` and ``ScaleFactor`` in the options a Scaled Background Subtraction will be
+Additional hidden columns can be shown on the table by checking the ``Scaled Background Subtraction`` checkbox above
+the table on the right hand side.
+
+By setting **both** ``Background Workspace`` and ``Scale Factor`` a Scaled Background Subtraction will be
```

Comment:
Is it worth making a note here about the fact that if the can values are in the row then we do sample minus can minus scaled background? When we were testing with the scientists, it sounded like this might be a good thing to highlight.